include/igraph_vector.h

```diff
@@ -136,6 +136,11 @@ __BEGIN_DECLS
     do { IGRAPH_CHECK(igraph_vector_bool_init(v, size)); \
         IGRAPH_FINALLY(igraph_vector_bool_destroy, v); } while (0)
 #endif
+#ifndef IGRAPH_VECTOR_CHAR_INIT_FINALLY
```

Comment:
Definitely useful - let's keep this.

Reply:
Would you like me to make a separate PR onto `develop` just with this? Happy to do it!

Reply:
PR done and merged